rave-compiler/src/test/resources/fixtures/longdef/BadLongDefUse.java

```diff
@@ -0,0 +1,24 @@
+package fixtures.longdef;
+
+import com.uber.rave.annotation.Validated;
+
+import fixtures.SampleFactory;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import android.support.annotation.LongDef;
+
+@Validated(factory = SampleFactory.class)
+public class BadLongDefUse {
```

Comment:
I think there is a test that already does that. @naturalwarren can you double check

Reply:
Test that covers that case is `RaveUnitTest.checkLongDef_whenValueDoesNotMatch_shouldProduceError`

Reply:
I meant in this model fixture

Reply:
Ah gotcha. Similar fixtures like `BadIntDefUse` haven't included that case in the past but it seems worthwhile since it would server as an end-to-end test that ensure the internals of RAVE are properly hooked up which I don't think we've covered anywhere else.

Reply:
Looking a little bit more closely we've covered this in `CompilerIntegrationTest.allInvalidLongDefModel_shouldFail()`. In this test construct a `LongDefModel` which returns a value that isn't part of the `LongDef`.